experimental/packages/shim-opencensus/package.json

```
@@ -65,10 +65,9 @@
     "@opentelemetry/api": "^1.0.0"
   },
   "dependencies": {
-    "@opentelemetry/context-async-hooks": "1.13.0",
-    "@opentelemetry/core": "1.13.0",
-    "require-in-the-middle": "^6.0.0",
-    "semver": "^7.3.5"
+    "@opentelemetry/core": "^1.0.0",
```

```suggestion
    "@opentelemetry/core": "1.13.0",
```
Within this repo/lerna project dependencies are pinned and automatically updated on release. As far as I know this is needed to get proper linking and to avoid mixture/duplicates of packages for end users

---

I have it pinned on L48 in dev dependencies. Things are working correct with lerna:

```sh
$ ls -l experimental/packages/shim-opencensus/node_modules/@opentelemetry/
total 0
lrwxrwxrwx 1 aaronabbott primarygroup 18 May 17 15:16 api -> ../../../../../api
lrwxrwxrwx 1 aaronabbott primarygroup 57 May 17 15:16 context-async-hooks -> ../../../../../packages/opentelemetry-context-async-hooks
lrwxrwxrwx 1 aaronabbott primarygroup 42 May 17 15:16 core -> ../../../../../packages/opentelemetry-core
lrwxrwxrwx 1 aaronabbott primarygroup 52 May 17 15:16 sdk-trace-base -> ../../../../../packages/opentelemetry-sdk-trace-base
```

> duplicates of packages for end users

If I leave the dependency more open, wouldn't that leave more room for npm to de-dup their dependencies? Alternatively if we want it to strictly be de-duplicated, maybe a peer dependency would be better here

---

Gonna resolve this, lets consolidate to the comment thread with Daniel.